vlib/v/checker/tests/is_type_invalid.out

```diff
@@ -5,10 +5,10 @@ vlib/v/checker/tests/is_type_invalid.vv:14:12: error: `IoS` has no variant `byte
       |               ~~
    15 |         println('not cool')
    16 |     }
-vlib/v/checker/tests/is_type_invalid.vv:18:5: error: `Cat` doesn't implement method `speak` of interface `Animal`
+vlib/v/checker/tests/is_type_invalid.vv:18:7: error: `Cat` doesn't implement method `speak` of interface `Animal`
    16 |     }
    17 |     a := Animal(Dog{})
    18 |     if a is Cat {
-      |        ~~~~~~~~
```

Comment:
I think the old position makes more sense 🤔

Reply:
Ok, I'll restore it to old position.